src/components/App/App.tsx

```diff
@@ -15,16 +15,16 @@ import View from '../View';
 import Hello from '../Hello';
 import { declareQueries } from 'avenger/lib/react';
 import { currentView } from '../../queries';
-
 import './app.scss';
+import * as config from './../../config';
 
 const queries = declareQueries({ currentView });
-
 class App extends React.Component<typeof queries.Props> {
   render() {
+    console.log(config);
     return (
       <View column className="app">
-        <h1>Yelp Searcher</h1>
+        <h1>Yelp Restaurant finder</h1>
```

Comment:
we try to use always only react components instead of html tags, I would replace the `<h1>` with a `<View classname='title'>some text</View>`